packages/utils/lib/index.ts

```diff
@@ -101,9 +101,9 @@ export const shuffle = <T>(array: Array<T>) => {
  * @name pageFromUri
  * @description Use url variables to load the default components upon the first page visit.
  */
-export const pageFromUri = (pathname: string) => {
+export const pageFromUri = (pathname: string, fallback?: string) => {
```

Comment:
Why is fallback optional? You're allowing the function to provide an empty fallback.